File: Algorithms/Sort Algorithms/Selection Sort/selection.js
const insert = (array) => {
    let n = array.length
    for (let i = 0; i < n; i++) {
        let smallestIndex = i
        for (let j = i + 1; j < n; j++) {
            if (array[j] < array[smallestIndex]) {
                smallestIndex = j
            }
        }
        [array[i], array[smallestIndex]] = [array[smallestIndex], array[i]]
    }
    return array
}

let testArray = [8, 5, 3, 2, 7, 1]
let sortedArray = insert(testArray)
console.log(sortedArray)

File: Algorithms/Sort Algorithms/Selection Sort/selection.py
def insert(array):
    n = len(array)
    for i in range(n):
        smallestIndex = i
        for j in range(i+1,n):
            if array[j] < array[smallestIndex]:
                smallestIndex = j
        array[i],array[smallestIndex] = array[smallestIndex],array[i]
    return array    

test_array = [6,87,3,2,1,0]
sorted_array = insert(test_array)
print(sorted_array)